### PR TITLE
Issue 67: change dependency to IO::Uncompress::Gunzip from PerlIO::gzip

### PR DESCRIPTION
**Pull request by zielot**
My update addresses the changes indicated by zzqzzqzzq's comments from Nov 9, 2017 suggesting changes to fix [issue 67](https://github.com/maxmind/geoipupdate/issues/67)

### CODE REVIEW

File: bin/geoipupdate-pureperl.pl [hunk 1]
```diff
@@ -257,7 +257,7 @@ sub _gunzip_and_replace {
         # --- uncompress the gzip data
         {
             local $_;
-            open my $gin,  '<:gzip', \$content  or die $!;
+	    my $gin = new IO::Uncompress::Gunzip( \$content ) or die $!;
```
Comment (oschwald) [inline bin/geoipupdate-pureperl.pl]: The indirect object syntax is fragile. Please change this to `IO::Uncompress::Gunzip->new` instead.

File: bin/geoipupdate-pureperl.pl [hunk 2]
```diff
@@ -49,7 +49,7 @@
 use Getopt::Std;
 use HTTP::Request::Common;
 use LWP::UserAgent;
-use PerlIO::gzip;
+use IO::Uncompress::Gunzip;
```
Comment (oschwald) [inline bin/geoipupdate-pureperl.pl]: Please alpha sort these.

**Review by oschwald — CHANGES_REQUESTED** [review]
Thanks for the change. I had a couple of small comments.

We may be removing the Perl script in the future. I am happy to merge a fixed version of this until then, but may I ask why you prefer the Perl version to the C one?

**Comment by zielot** [comment]
Hi Gregory,

It would be a shame if you were to get rid of the Perl script. The
difference in complexity reminds me of why I stopped developing in C.
However, the reason why we prefer the Perl script is simple: we are a Java
shop and do not have a C development environment set up anywhere whereas
Perl is ubiquitous. If you were to provide binaries or even better, rpms
then this would not be an issue.

~ Justin

**Comment by oschwald** [comment]
Thanks!

**Pull request merged**